kpconv_torch/test.py

```diff
@@ -223,7 +223,7 @@ def main(args):
     if config.dataset_task == "classification":
         tester.classification_test(net, test_loader, config)
     elif config.dataset_task == "cloud_segmentation":
-        tester.cloud_segmentation_test(net, test_loader, config)
+        tester.cloud_segmentation_test(net, test_loader, config, args.filename, args.chosen_log)
```

Comment:
Made in another PR in an other way, to treat when merging

Reply:
Rebasing on this other branch could be an alternative as well.